server/routerlicious/packages/services-core/src/mongoDatabaseManager.ts

```diff
@@ -31,6 +32,10 @@ export class MongoDatabaseManager implements IDatabaseManager {
 		return this.getCollection<IDocument>(this.documentsCollectionName);
 	}
 
+	public async getCheckpointCollection(): Promise<ICollection<ICheckpoint>> {
```

Comment:
If this method not used anywhere, no need to place this to the database manager

Reply:
@zhangxin511, this method is used in the localOrdererSetup